@ndswartz thanks for updating the Ansible docs. 

Now that you've fixed a mismatch between the documentation and the argspec, you need to remove this module from the list of mismatches to ignore ([code here](https://github.com/ansible/ansible/blob/devel/test/sanity/validate-modules/ignore.txt#L3459)) - until you do, this PR will fail CI.

Full error message is:
```
 test/sanity/validate-modules/ignore.txt:3459:1: A102 
Remove since "lib/ansible/modules/storage/netapp/netapp_e_volume_copy.py" passes "E338" test
```

--- a/lib/ansible/plugins/doc_fragments/netapp.py
+++ b/lib/ansible/plugins/doc_fragments/netapp.py
@@ -163,14 +163,17 @@ notes:
 options:
   api_username:
     required: true
+    type: str
     description:
     - The username to authenticate with the SANtricity Web Services Proxy or Embedded Web Services API.
   api_password:
     required: true
+    type: str
     description:
     - The password to authenticate with the SANtricity Web Services Proxy or Embedded Web Services API.
   api_url:
     required: true
+    type: str
     description:
     - The url to the SANtricity Web Services Proxy or Embedded Web Services API.
       Example https://prod-1.wahoo.acme.com/devmgr/v2
@@ -182,6 +185,7 @@ options:
     type: bool
   ssid:
     required: false
+    type: str
     default: 1
     description:
     - The ID of the array to manage. This value must be unique for each array.

--- a/test/sanity/validate-modules/ignore.txt
+++ b/test/sanity/validate-modules/ignore.txt
@@ -3516,7 +3516,6 @@ lib/ansible/modules/storage/netapp/netapp_e_volume_copy.py E324
 lib/ansible/modules/storage/netapp/netapp_e_volume_copy.py E326
 lib/ansible/modules/storage/netapp/netapp_e_volume_copy.py E335
 lib/ansible/modules/storage/netapp/netapp_e_volume_copy.py E337
-lib/ansible/modules/storage/netapp/netapp_e_volume_copy.py E338
 lib/ansible/modules/storage/netapp/netapp_e_volume.py E324
 lib/ansible/modules/storage/netapp/netapp_e_volume.py E327
 lib/ansible/modules/storage/netapp/netapp_e_volume.py E337

